server/container/guice/pom.xml

```diff
@@ -37,21 +37,8 @@
 
     <modules>
         <module>guice-common</module>
+        <module>cassandra-guice</module>
+        <module>memory-guice</module>
     </modules>
-
-    <profiles>
-        <profile>
```

Comment:
If you don't give the Cassandra profile to maven, the `SmtpTestModule` (in mpt-smtp-cassandra) won't compile as you will not satisfy the james-server-cassandra-guice dependency.